pkgs/stdenv/generic/default.nix

```diff
@@ -223,7 +223,7 @@ let
         # include it in the result, it *is* available to nix-env for queries.
         meta = { }
             # If the packager hasn't specified `outputsToInstall`, choose a default,
-            # namely `p.bin or p.out or p`;
+            # namely `lib.getBin p`;
```

Comment:
Hm, what do you mean by "default output", `pkg`? If so I cannot see how can it be chosen there, because we iterate over some list and try each _attribute_ without ever resorting to the attribute set itself. If nothing was found, we'd just return `[null]`.